VKApp/VKApp/Models/Posts/Post.swift

```diff
@@ -0,0 +1,25 @@
+// Post.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Posts
+final class Posts: Decodable {
+    var items: [PostItems]?
+    var group: [Groups]?
+    var profile: [Item]?
+
+    enum CodingKeys: String, CodingKey {
+        case items
+        case group = "groups"
+        case profile = "profiles"
```

Comment:
```suggestion
        case profiles
```
Groups profiles массивы